docs/dg/dev/architecture/api-platform.md

```diff
@@ -0,0 +1,426 @@
+---
+title: API Platform
+description: Spryker's API Platform integration provides schema-based API resource generation with automatic OpenAPI documentation.
+last_updated: Nov 24, 2025
+template: concept-topic-template
+related:
+  - title: How to integrate API Platform
+    link: docs/dg/dev/upgrade-and-migrate/integrate-api-platform.html
+  - title: Dependency Injection
+    link: docs/dg/dev/architecture/dependency-injection.html
+---
+
+Spryker's API Platform integration provides schema-based API resource generation with automatic OpenAPI documentation. This allows you to define your API resources using YAML or XML schemas and automatically generate fully-functional API endpoints with validation, pagination, and serialization.
```

Comment:
Would you like to add from the beginning, that this is https://api-platform.com/ that we use behind, so readers can catch the "Aha!" right from the start?